### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: src/z3c/wizard/interfaces.py [hunk 1]
```diff
@@ -239,18 +235,15 @@ class IWizard(zope.location.interfaces.ILocation):
         required=False)
 
     def doAdjustStep():
-        """Ensure that we can't traverse to more then the first not completed 
-        step.
+        """Ensure traversal is only possible up to first not completed step.
+
+        Trying to traverse to an other step redirects to the first incomplete
```
Comment (mgedmin) [inline src/z3c/wizard/interfaces.py]: Typo: an other -> another

File: src/z3c/wizard/zcml.py [hunk 2]
```diff
@@ -110,20 +99,18 @@ class IWizardStepDirective(zope.interface.Interface):
...
         required=False,
-        value_type=zope.configuration.fields.GlobalInterface(),
-        )
+        value_type=zope.configuration.fields.GlobalInterface())
 
     allowed_attributes = zope.configuration.fields.Tokens(
-        title=u"View attributes that are also allowed if the user"
-                " has permission.",
+        title=u"View attributes that are also allowed if the user "
+              u"has permission.",
```
Comment (mgedmin) [inline src/z3c/wizard/zcml.py]: Personally I prefer the blank to be on the 2nd line -- makes it easier to visually check that it's not missing.

**Review by mgedmin — APPROVED** [review]
(Looks like the flake8 changes are intermixed with Python 3 porting changes.)

LGTM anyway.

File: src/z3c/wizard/zcml.py (hunk 2 above)
Comment (icemac) [reply src/z3c/wizard/zcml.py]: I've never seen this before but is sounds reasonable.

**Pull request merged**